add(translation): add basic translation

igorsaux opened the pull request:
Добавлена поддержка локализации на базе https://projectfluent.org.

Локализация осуществляется с помощью динамической библиотеки https://github.com/igorsaux/fluent-onyx.

Любой файл в папке `localizations/` с расширением `.ftl` расценивается как отдельный язык.

close #6723
close #6722

<details>
<summary>Чейнджлог</summary>

```yml
🆑
rscadd: Добавлена поддержка локализации.
spellcheck: Переведены анонсы на русский язык.
/🆑
```

</details>

- [ ] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Doster-d commented:
О, бенарнеке

Zert0X commented:
Оно проклято, бедная читаемость кода.

igorsaux commented:
Осталось перевести

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $station_name }
+
+announce-meteors-end =
+    Метеоритный шторм пролетел { $station_name }.
```
Я бы так сформулировал

Метеорный шторм прошел территорию станции.

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Level X
+
+announce-level-x =
+    Подтверждена вспышка биологической опасности уровня { $bio_level } на борту { $station_name }.
```
биологической угрозы

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-level-x =
+    Подтверждена вспышка биологической опасности уровня { $bio_level } на борту { $station_name }.
+    Весь персонал должен сдерживать вспышку.
```
Можно так:

Всему персоналу необходимо сдержать угрозу.

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Весь персонал должен сдерживать вспышку.
+
+announce-level-x-title =
+    Предупреждение о биологической опасности
```
угрозе

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-ion-storm =
+    До нашего сведения дошло, что { $station_name } прошел через ионную бурю.
+    Пожалуйста, следите за всем электронным оборудованием на предмет неисправностей.
```
проверьте все электронное оборудование

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Grid Restored
+
+announce-grid-restored =
+    На этот раз питание станции на { $station_name } будет восстановлено.
```
Питание было восстановлено на станции { $station_name }

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Bluespace Artillery
+
+announce-bluespace-artillery-fire =
+    Обнаружен огонь артиллерией синего пространства. Приготовьтесь к удару.
```
блюспейс артиллерии, имя собственное

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $station_name }
+
+announce-meteors-end =
+    Метеорный шторм прошел территорию станции.
```
Либо
```suggestion
    Метеоритный шторм прошел территорию станции.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $station_name }
+
+announce-meteors-end-minor =
+    Метеоритный дождь прошёл { $station_name }.
```
Либо
```suggestion
    Метеорный дождь прошёл { $station_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Unidentified Lifesigns
+
+announce-unidentified-lifesigns =
+    Обнаружены неопознанные признаки жизни, поступающие на борт { $station_name }.
```
```suggestion
    Обнаружены неопознанные признаки жизни на борту { $station_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-grey-tide =
+    Вирус Gr3y.T1d3 обнаружен в подпрограммах тюремного заключения { $station_name }.
+    Рекомендую задействовать искусственный интеллект.
```
```suggestion
    Рекомендуется задействовать искусственный интеллект.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-electrical-storm-major =
+    Тревога. Сильная электрическая буря была обнаружена в непосредственной близости от { $location_name }.
+    Рекомендуется немедленно обезопасить чувствительное электрооборудование до тех пор, пока шторм не пройдет.
```
```suggestion
    Рекомендуется немедленно обезопасить чувствительного электрооборудования до тех пор, пока шторм не пройдет.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Brand Intelligence
+
+announce-brand-intelligence =
+    На борту { $location_name } был обнаружен безумный корпоративный интеллект.
```
```suggestion
    На борту { $location_name } был обнаружен аномальный искусственный интеллект.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Предупреждение о машинном обучении
+
+announce-brand-intelligence-end =
+    Все следы безумного корпоративного интеллекта исчезли из систем.
```
```suggestion
    Все следы аномального искусственного интеллекта исчезли из систем.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Tear Reality
+
+announce-tear-reality =
+    Высокие уровни помех синего пространства обнаружены в { $area }.
```
```suggestion
    Высокие уровни помех блюспейса обнаружены в { $area }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-tear-reality =
+    Высокие уровни помех синего пространства обнаружены в { $area }.
+    Подозревается образование червоточины. Немедленно расследуйте это.
```
```suggestion
    Ожидается образование червоточин. Немедленно обнаружьте их.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Подозревается образование червоточины. Немедленно расследуйте это.
+
+announce-tear-reality-ceased =
+    Аномалия синего пространства прекратилась.
```
```suggestion
    Блюспейс помехи прекратились.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-supermatter-cascade =
+    Был обнаружен общегалактический электромагнитный импульс. Все системы в космосе сильно повреждены, и
+    многие сотрудники погибли или умирают. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
```
```suggestion
    многие сотрудники пострадали или погибли. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    многие сотрудники погибли или умирают. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
+
+    { $station_name }, самый большой источник помех точно указывает непосредственно на вас.
+    По нашим оценкам, у вас есть пять минут до того, как в ваших помещениях откроется разлом синего пространства.
```
```suggestion
    По нашим оценкам, у вас есть пять минут до того, как в ваших помещениях откроется разлом блюспейса.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Не существует известного способа остановить образование разлома, равно как и какого-либо способа избежать его.
+    Вы совершенно одни.
+
+    Да спасёт Господь ваши з[\[###!!!-
```
```suggestion
    Да спасёт Господь ваши д[\[###!!!-
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+    Да спасёт Господь ваши з[\[###!!!-
+
+    АВТОМАТИЧЕСКОЕ ОПОВЕЩЕНИЕ: Ссылка на { $command_name } потеряна.
```
```suggestion
    АВТОМАТИЧЕСКОЕ ОПОВЕЩЕНИЕ: Связь с { $command_name } была потеряна.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    АВТОМАТИЧЕСКОЕ ОПОВЕЩЕНИЕ: Ссылка на { $command_name } потеряна.
+
+announce-supermatter-cascade-title =
+    ОБНАРУЖЕН КАСКАД СВЕРХМАТЕРИИ
```
```suggestion
    ОБНАРУЖЕН КАСКАД СУПЕРМАТЕРИИ
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Gateway Distress
+
+announce-gateway-distress =
+    Обнаружена аномалия безопасности в { $gateway_name }{ $location }. Попытка установить изо... Ошибка.
```
```suggestion
    Обнаружена проблема в безопасности { $gateway_name }{ $location }. Попытка изолир... ОШИБКА.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    IDS сообщает, что IPS был отключен. Уровень тревоги о вторжении: КРАСНЫЙ.
+
+    Идентификатор сообщает, что передача была принята устройством. Источник передачи "Неизвестен",
+    подпись: "Сигнал бедствия высокого уровня NT". Обнаружен процесс триангуляции.
```
```suggestion
    Подпись: "Сигнал бедствия NT высшего уровня". Обнаружен процесс триангуляции.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Обнаружена аномалия безопасности в { $gateway_name }{ $location }. Попытка установить изо... Ошибка.
+    IDS сообщает, что IPS был отключен. Уровень тревоги о вторжении: КРАСНЫЙ.
+
+    Идентификатор сообщает, что передача была принята устройством. Источник передачи "Неизвестен",
```
```suggestion
    Идентификатор сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Gravity
+
+announce-gravity =
+    Всплеск обратной связи обнаружен в системах массового распределения.
```
```suggestion
    Всплеск обратной гравитационной связи обнаружен в системах массового распределения.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Infestation
+
+announce-infestation =
+    Биосканы показывают, что { $vermstring } размножались в { $location }.
```
```suggestion
    Биосканеры показывают, что { $vermstring } размножились в { $location }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-infestation =
+    Биосканы показывают, что { $vermstring } размножались в { $location }.
+    Очистите их, прежде чем это начнет влиять на производительность.
```
```suggestion
    Разберитесь с ними, прежде чем это начнет влиять на производительность.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Money Hacker
+
+announce-money-hacker =
+    Обнаружен взлом методом перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета,
```
```suggestion
    Обнаружен взлом с помощью метода перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-money-hacker =
+    Обнаружен взлом методом перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета,
+    без вмешательства эта атака завершится успешно примерно через 10 минут. Возможные решения: приостановка учетных записей, отключение интернет-сервера,
```
```suggestion
    Без вмешательства эта атака успешно завершится примерно через 10 минут. Возможные решения: приостановка учетных записей, отключение интернет-сервера,
```

kreeperHLC commented on localization/en.ftl:
```diff
@@ -0,0 +1,719 @@
+
+announce-money-hacker =
+    A brute force hack has been detected (in progress since { $time }). The target of the attack is: Financial accounts,
+    without intervention this attack will succeed in approximately 10 minutes. Possible solutions: suspension of accounts, disabling NTnet server,
```
```suggestion
    A brute force hack has been detected (in progress since { $time }). The target of the attack is: Financial accounts.
    Without intervention this attack will succeed in approximately 10 minutes. Possible solutions: suspension of accounts, disabling NTnet server,
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Попытка взлома увенчалась успехом.
+
+announce-money-hacker-loose =
+    Атака прекратилась, теперь затронутые учетные записи можно вывести в Интернет.
```
```suggestion
    Атака прекратилась, затронутые учетные записи в безопасности.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-ictus =
+    На станции была замечена подозрительная биологическая активность.
+    Медицинская бригада должна немедленно подготовиться к борьбе с возбудителем.
```
```suggestion
    Медицинская бригада должна немедленно подготовиться к борьбе с возбудителем вируса.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+announce-prison-break =
+    { PICK("Вирус Gr3y.T1d3", "Злокачественный троян") } обнаружен в подпрограммах { $location_name } { $type }.
+    Немедленно защитите все скомпрометированные области.
+    рhекомендуется использовать искусственный интеллект { $location_name }.
```
```suggestion
    рекомендуется использовать искусственный интеллект { $location_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+announce-radiation-storm-passed =
+    { $location_name } прошел радиационный пояс.
+    Пожалуйста, подождите до одной минуты, пока уровень радиации рассеется,
+    и сообщите в лазарет, если у вас возникнут какие-либо необычные симптомы.
```
```suggestion
    и сообщите в медбей, если у вас возникнут какие-либо необычные симптомы.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    { $location_name } прошел радиационный пояс.
+    Пожалуйста, подождите до одной минуты, пока уровень радиации рассеется,
+    и сообщите в лазарет, если у вас возникнут какие-либо необычные симптомы.
+    Техническое обслуживание вскоре снова потеряет весь доступ.
```
```suggestion
    Персонал вскоре снова потеряет доступ в туннели технического обслуживания.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Неопознанные беспилотные летательные аппараты приближаются к { $location_name }.
+
+announce-rogue-drone-03 =
+    Лазерный огонь класса II обнаружен рядом с { $location_name }.
```
```suggestion
    Лазерный дрон класса II обнаружен рядом с { $location_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $location_name }
+
+announce-rogue-drone-end =
+    Имейте в виду: датчики показывают, что неопознанный рой дронов покинул непосредственную близость к { $location_name }.
```
```suggestion
    Имейте в виду: датчики показывают, что неопознанный рой дронов покинул  { $location_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-solar-storm =
+    Была обнаружена солнечная буря, приближающаяся к { $location_name }.
+    Пожалуйста, немедленно прекратите все ВКД и вернитесь внутрь.
```
```suggestion
    Пожалуйста, немедленно прекратите любую деятельность в космосе и вернитесь внутрь.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $location_name }
+
+announce-solar-storm-start =
+    Солнечная буря достигла { $location_name }. Пожалуйста, воздержитесь от ВКД и оставайтесь внутри, пока она не пройдет.
```
```suggestion
    Солнечная буря достигла { $location_name }. Пожалуйста, воздержитесь любой космической деятельности и оставайтесь внутри, пока она не пройдет.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Солнечная буря достигла { $location_name }. Пожалуйста, воздержитесь от ВКД и оставайтесь внутри, пока она не пройдет.
+
+announce-solar-storm-end =
+    Солнечная буря прошла { $location_name }. Теперь можно безопасно возобновить деятельность ВКД.
```
```suggestion
    Солнечная буря прошла { $location_name }. Теперь можно возобновить космическую деятельность.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Wallrot
+
+announce-wallrot =
+    Злокачественные грибы обнаружены в { $location_name }.
```
```suggestion
    Злокачественный настенный грибок обнаружен в { $location_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Power Restore
+
+announce-power-restore =
+    Все СПИНЭ на { $station_name } были пополнены. Приносим извинения за причиненные неудобства.
```
```suggestion
    Все СМЕСы на { $station_name } были пополнены. Приносим извинения за причиненные неудобства.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+announce-hack-failure-02 =
+    Мы обнаружили еще одну попытку взлома.
+    Он был нацелен на { $text }. Злоумышленник почти получил контроль над системой,
+    поэтому нам пришлось отключить их. Мы частично завершили нашу трассировку, и, похоже,
```
```suggestion
    поэтому нам пришлось её отключить. Мы частично завершили нашу трассировку, и, похоже,
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Похоже, кто-то пытается взломать ваши электронные системы. Мы сообщим вам, когда у нас будет больше информации.
+
+announce-hack-progress-02 =
+    Мы начали выслеживать злоумышленника. Кто бы это ни делал, они, похоже, на борту.
```
```suggestion
    Мы начали выслеживать злоумышленника. Кто бы это ни делал, он, похоже, находится на борту.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    он ускользает от наших следов. Ни один человек не мог быть столь быстрым...
+
+announce-hack-progress-04 =
+    Мы обнаружили злоумы#, это похо&е ( Ъ в4ш И1, 0но &# взл*#@ет $и$тему $ам0 ун14т0#ен1я,
```
```suggestion
    Мы обнаружили злоумы#, это похо&е ( Ъ в4ш И1, 0н &# взл*#@ет $и$тему $ам0 ун14т0#ен1я,
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-hack-progress-04 =
+    Мы обнаружили злоумы#, это похо&е ( Ъ в4ш И1, 0но &# взл*#@ет $и$тему $ам0 ун14т0#ен1я,
+    0$тано81те ег# п0ка 0н не*@!)$#&&@@  <CONNECTION LOST>
```
```suggestion
    0$тано81те ег# п0ка 0н не*@!)$#&&@@  <ПОТЕРЯ СВЯЗИ>
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Был отправлен аварийный спасательный шаттл.
+
+emergency-huttle-called-demeter =
+    Для этого объекта был заказан экстренный запрос.
```
```suggestion
    Для этого объекта был отправлен экстренный запрос.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Evacuation Canceled
+
+announce-bluespace-distortion-cancels-evacuation =
+    Эвакуация была прервана из-за искажения синего пространства.
```
```suggestion
    Эвакуация была прервана из-за блюспейс искажения.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Arrival Message
+
+announce-arrival-message-syndicate =
+    Внимание, у вас есть большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования.  
```
```suggestion
    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования.  
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-arrival-message-syndicate =
+    Внимание, у вас есть большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования.  
+    Мы слишком далеко, чтобы перехватить - приготовьтесь к визитерам.
```
```suggestion
    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Мы слишком далеко, чтобы перехватить - приготовьтесь к визитерам.
+
+announce-arrival-message-skipjack =
+    Внимание, у вас есть большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования.  
+    Мы слишком далеко, чтобы перехватить - приготовьтесь к визитерам.
```
```suggestion
    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования. 
    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Departure Message
+
+announce-departure-message-syndicate =
+    Ваши посетители на пути к выходу из системы, сжигая дельту-v, как будто это ничего не значит. Скатертью дорожка.
```
```suggestion
    Ваши посетители на пути к выходу из системы, удирая так быстро, как только можно. Скатертью дорожка.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Ваши посетители на пути к выходу из системы, сжигая дельту-v, как будто это ничего не значит. Скатертью дорожка.
+
+announce-departure-message-skipjack =
+    Ваши посетители на пути к выходу из системы, сжигая дельту-v, как будто это ничего не значит. Скатертью дорожка.
```
```suggestion
    Ваши посетители на пути к выходу из системы, удирая так быстро, как только можно. Скатертью дорожка.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Borer Spawn
+
+announce-borer-spawn =
+    Обнаружены неопознанные признаки жизни, поступающие на борт { $station_name }.
```
```suggestion
    Обнаружены неопознанные признаки жизни, поступающие с борта { $station_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Alien Spawn
+
+announce-alien-spawn =
+    Обнаружены неопознанные признаки жизни, поступающие на борт { $station_name }.
```
```suggestion
    Обнаружены неопознанные признаки жизни, поступающие с борта { $station_name }.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+# Security Levels
+
+announce-security-code-elevated-title =
+    Внимание! Уровень оповещения повышен до { $name }!
```
```suggestion
    Внимание! Уровень безопасности повышен до { $name }!
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Внимание! Уровень оповещения повышен до { $name }!
+
+announce-security-code-changed-title =
+    Внимание! Уровень оповещения изменен на { $name }!
```
```suggestion
    Внимание! Уровень безопасности изменен на { $name }!
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Code Green
+
+announce-security-code-green-down =
+    Все угрозы станции миновали. У службы безопасности может не быть видимого оружия,
```
```suggestion
    Все угрозы станции миновали. Службы безопасности не может носить оружие на виду,
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-security-code-blue-up =
+    Станция получила достоверную информацию о возможной враждебной деятельности на станции.
+    У сотрудников службы безопасности может быть видимое оружие, разрешены случайные обыски".
```
```suggestion
    У сотрудников службы безопасности может носить оружие на виду, разрешены "случайные обыски".
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-security-code-blue-down =
+    Непосредственная угроза миновала. Служба безопасности больше не может постоянно держать оружие наготове,
+    но может продолжать держать его на виду. Случайный поиск по-прежнему разрешен.
```
```suggestion
    но может продолжать носить его на виду. Случайный поиск по-прежнему разрешен.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-security-code-red-up =
+    Существует непосредственная серьезная угроза для станции.
+    Служба безопасности может постоянно иметь оружие без кобур.
```
```suggestion
    Служба безопасности может постоянно носить оружие без кобуры.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-security-code-red-down =
+    Механизм самоуничтожения был деактивирован, однако непосредственная серьезная угроза станции по-прежнему существует.
+    Служба безопасности может постоянно иметь оружие без кобур, разрешены и рекомендованы случайные обыски.
```
```suggestion
    Служба безопасности может постоянно носить оружие без кобуры, разрешены и рекомендованы случайные обыски.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-response-team-call-failed =
+    Похоже, что для { $station_name } была запрошена группа экстренного реагирования.
+    К сожалению, в настоящее время мы не смогли отправить ни одного из них.
```
```suggestion
    К сожалению, в настоящее время мы не смогли отправить никого из её состава.
```

kreeperHLC commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-response-team-called =
+    Похоже, что для { $station_name } была запрошена группа экстренного реагирования.
+    Мы подготовим и отправим его как можно скорее.
```
```suggestion
    Мы подготовим и отправим её как можно скорее.
```

TobyThorne commented on code/modules/events/rogue_drones.dm:
```diff
@@ -40,9 +44,9 @@
 		qdel(D)
-		num_recovered++
 
-	if(num_recovered > drones_list.len * 0.75)
```
а поч тут логика померла

igorsaux replied:
В обоих случаях выполняется одинаковый анонс

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-level-x =
+    Подтверждена вспышка биологической угрозы уровня { $bio_level } на борту { $station_name }.
+    Всему персоналу необходимо сдержать угрозу.
```
```suggestion
    Весь персонал обязан соучаствовать в сдерживании угрозы.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Unidentified Lifesigns
+
+announce-unidentified-lifesigns =
+    Обнаружены неопознанные признаки жизни на борту { $station_name }.
```
```suggestion
    Обнаружено присутствие неопознанных организмов на борту { $station_name }.
```
Лучше уж так, чем _неопознанные признаки жизни_.

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Radiation Detected
+
+announce-radiation-detected =
+    Высокий уровень радиации был обнаружен в непосредственной близости от { $station_name }.
```
```suggestion
    Высокий уровень радиации был зафиксирован в непосредственной близости от { $station_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    В вашем районе была обнаружена электрическая буря, пожалуйста, устраните возможные электронные перегрузки.
+
+announce-electrical-storm-title =
+    Предупреждение о электрическом шторме
```
```suggestion
    Предупреждение об электрическом шторме
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Electrical Storm
+
+announce-electrical-storm =
+    В вашем районе была обнаружена электрическая буря, пожалуйста, устраните возможные электронные перегрузки.
```
```suggestion
    В вашем районе была обнаружена электрическая буря, пожалуйста, устраните возможные электрические перегрузки.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Пожалуйста, устраните любые электрические перегрузки.
+
+announce-electrical-storm-end-title =
+    Предупреждение о электрическом шторме
```
```suggestion
    Предупреждение об электрическом шторме
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Brand Intelligence
+
+announce-brand-intelligence =
+    На борту { $location_name } был обнаружен аномальный искусственный интеллект.
```
```suggestion
    На борту { $location_name } было обнаружено зловредное маркетинговое ПО.
```
Ближе по смыслу, хотя всё ещё не совсем дословно. Зато смешно.

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Предупреждение о машинном обучении
+
+announce-brand-intelligence-end =
+    Все следы аномального искусственного интеллекта исчезли из систем.
```
```suggestion
    Все следы зловредного ПО исчезли из систем.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Tear Reality
+
+announce-tear-reality =
+    Высокие уровни помех блюспейса обнаружены в { $area }.
```
```suggestion
    Высокие уровни блюспейс-интерференции обнаружены в { $area }.
```
То же самое, но заумнее. Ибо блюспейс.

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Ожидается образование червоточин. Немедленно обнаружьте их.
+
+announce-tear-reality-ceased =
+    Блюспейс помехи прекратились.
```
```suggestion
    Блюспейс-интерференция возвращается к безопасным значениям.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-supermatter-cascade =
+    Был обнаружен общегалактический электромагнитный импульс. Все системы в космосе сильно повреждены, и
+    многие сотрудники мертвы или скоро умрут. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
```
```suggestion
    многие сотрудники либо уже мертвы, либо серьёзно пострадали. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Не существует известного способа остановить образование разлома, равно как и какого-либо способа избежать его.
+    Вы совершенно одни.
+
+    Да спасёт Господь ваши д[\[###!!!-
```
```suggestion
    Да помилует Господь ваши д[\[###!!!-
```

TobyThorne replied:
Тут для "спасёт" уже поздновато

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $location_name }
+
+announce-dust-end =
+    { $location_name } прошел через пояс космической пыли.
```
```suggestion
    { $location_name } покидает пояс космической пыли.
```
А то вдруг будет станция с женским именем

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Infestation
+
+announce-infestation =
+    Биосканеры показывают, что { $vermstring } размножились в { $location }.
```
```suggestion
    Биосканеры показывают, что { $vermstring } размножаются в { $location }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Разберитесь с ними, прежде чем это начнет влиять на производительность.
+
+announce-infestation-title =
+    Датчик морских тварей майора Билла
```
```suggestion
    Датчик корабельных крыс Майора Билла
```

TobyThorne replied:
А вообще тут можно и оригинальное название оставить, всё таки слишком уж название.

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Money Hacker
+
+announce-money-hacker =
+    Обнаружен взлом с помощью метода перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета.
```
```suggestion
    Зафиксирована брутфорс-атака (выполняется с момента { $time }). Целью атаки являются: Финансовые счета.
```
А то чёт совсем **_перебор_**

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+announce-money-hacker =
+    Обнаружен взлом с помощью метода перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета.
+    Без вмешательства эта атака успешно завершится примерно через 10 минут. Возможные решения: приостановка учетных записей, отключение интернет-сервера,
+    повышение уровня безопасности учетной записи. Уведомления будут отправляться по мере появления обновлений.
```
```suggestion
    повышение уровня безопасности учетных записи. Уведомления будут отправляться по мере появления обновлений.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-money-hacker =
+    Обнаружен взлом с помощью метода перебора (выполняется с момента { $time }). Целью атаки являются: Финансовые счета.
+    Без вмешательства эта атака успешно завершится примерно через 10 минут. Возможные решения: приостановка учетных записей, отключение интернет-сервера,
```
```suggestion
    Без вмешательства эта атака успешно завершится примерно через 10 минут. Возможные решения: временная блокировка учетных записей, отключение НТнет-сервера,
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-ictus =
+    На станции была замечена подозрительная биологическая активность.
+    Медицинская бригада должна немедленно подготовиться к борьбе с возбудителем вируса.
```
```suggestion
    Медицинский персонал должен немедленно подготовиться к борьбе с возбудителем вируса.
```
https://ru.wikipedia.org/wiki/%D0%9C%D0%B5%D0%B4%D0%B8%D1%86%D0%B8%D0%BD%D1%81%D0%BA%D0%B8%D0%B9_%D0%BF%D0%B5%D1%80%D1%81%D0%BE%D0%BD%D0%B0%D0%BB

No-Tips replied:
Не знаю кого как, но меня вирус возбудил

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Массив датчиков { $location_name }
+
+announce-radiation-storm-entered =
+    Объект { $location_name } вошел в радиационный пояс.
```
```suggestion
    Объект { $location_name } входит в радиационный пояс.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Пожалуйста, оставайтесь в защищенном месте, пока мы не пройдем радиационный пояс.
+
+announce-radiation-storm-passed =
+    { $location_name } прошел радиационный пояс.
```
```suggestion
    { $location_name } покидает радиационный пояс.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-radiation-storm-passed =
+    { $location_name } прошел радиационный пояс.
+    Пожалуйста, подождите до одной минуты, пока уровень радиации рассеется,
```
```suggestion
    Пожалуйста, подождите до одной минуты, пока уровень радиации спадёт,
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Rogue Drone
+
+announce-rogue-drone-01 =
+    Внимание: неопознанные патрульные беспилотники обнаружены в непосредственной близости от { $location_name }
```
```suggestion
    Внимание: неопознанные патрульные дроны обнаружены в непосредственной близости от { $location_name }
```
Что-что, а дрон уж точно слишком плотно вжился в русский язык
Компьютеры всё таки ЭВМами в 2к22 не называем

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Внимание: неопознанные патрульные беспилотники обнаружены в непосредственной близости от { $location_name }
+
+announce-rogue-drone-02 =
+    Неопознанные беспилотные летательные аппараты приближаются к { $location_name }.
```
```suggestion
    Неопознанные беспилотные дроны приближаются к { $location_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Неопознанные беспилотные летательные аппараты приближаются к { $location_name }.
+
+announce-rogue-drone-03 =
+    Лазерный дрон класса II обнаружен рядом с { $location_name }.
```
```suggestion
    Боевой дрон Class II Laser Fire обнаружен рядом с { $location_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Viral Outbreak
+
+announce-viral-outbreak =
+    Подтверждена вспышка биологической опасности 7-го уровня на борту { $location_name }.
```
```suggestion
    Подтверждена вспышка биологической угрозы 7-го уровня на борту { $location_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-viral-outbreak =
+    Подтверждена вспышка биологической опасности 7-го уровня на борту { $location_name }.
+    Весь персонал должен сдерживать вспышку.
```
```suggestion
    Весь персонал обязан соучаствовать в сдерживании угрозы.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Весь персонал должен сдерживать вспышку.
+
+announce-viral-outbreak-title =
+    Предупреждение о биологической опасности
```
```suggestion
    Предупреждение о биологической угрозе
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Конструкции могут быть заражены.
+
+announce-wallrot-title =
+    Предупреждение о биологической опасности
```
```suggestion
    Предупреждение о биологической угрозе
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Wormholes
+
+announce-wormholes =
+    На станции обнаружены пространственно-временные аномалии.
```
```suggestion
    На борту станции обнаружены пространственно-временные аномалии.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Immovable Rod
+
+announce-immovable-rod =
+    Что, черт возьми, это было?!
```
```suggestion
    Что это, блядь, было?!
```
Кажется, ИРЛ Игорь кино переводит

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Power Restore
+
+announce-power-restore =
+    Все СПИНЭ на { $station_name } были пополнены. Приносим извинения за причиненные неудобства.
```
```suggestion
    Все СМЕСы на { $station_name } были пополнены. Приносим извинения за причиненные неудобства.
```
Все уж слишком привыкли к СМЕСам, а от такого варианта у меня аж боли в СПИНЭ начинаются

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Shuttle Called
+
+announce-shuttle-called-exodus =
+    Запланирован трансфер экипажа с { $dock_name }.
```
А что если "развозка" вместо "трансфера"? :kek:

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        [one] { $eta } минуту
+        [few] { $eta } минуты
+        *[many] { $eta } минут
+    } шаттл приоетит в { $dock_name }.
```
```suggestion
    } шаттл прилетит в { $dock_name }.
```
приоетели

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Emergency Shuttle Call
+
+announce-emergency-shuttle-called-exodus =
+    Был вызван аварийный эвакуационный шаттл. Он прибудет примерно через { $eta ->
```
```suggestion
    Был вызван экстренный эвакуационный шаттл. Он прибудет примерно через { $eta ->
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    }.
+
+announce-emergency-shuttle-called-example =
+    Был отправлен аварийный спасательный шаттл.
```
```suggestion
    Был отправлен экстренный спасательный шаттл.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Emergency Shuttle Recall
+
+announce-emergency-shuttle-recall-exodus =
+    Аварийный шаттл был отозван.
```
```suggestion
    Экстренный шаттл был отозван.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Аварийный шаттл был отозван.
+
+announce-emergency-shuttle-recall-example =
+    Аварийный шаттл был отозван.
```
```suggestion
    Экстренный шаттл был отозван.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Аварийный шаттл был отозван.
+
+announce-emergency-shuttle-recall-demeter =
+    Наблдюдающий отклонил экстренный запрос. Возвращайтесь на свой пост.
```
```suggestion
    Наблюдающий отклонил экстренный запрос. Возвращайтесь на свои рабочие места.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Emergency Shuttle Docked
+
+announce-emergency-shuttle-docked-exodus =
+    Аварийный шаттл состыковался со станцией.
```
```suggestion
    Экстренный шаттл состыковался со станцией.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    У вас есть примерно { $etd -> 
+        [one] { $etd } минута
+        *[many] { $etd } минуты
+    } для посадки на Аварийный шаттл.
```
```suggestion
    } для посадки на экстренный шаттл.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    } для посадки на Аварийный шаттл.
+
+announce-emergency-shuttle-docked-example =
+    Аварийный спасательный шаттл состыкаовался.
```
```suggestion
    Экстренный спасательный шаттл состыковался.
```
Лютик, блядь!

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Emergency Shuttle Leaving Dock
+
+announce-emergency-shuttle-leaving-dock-exodus =
+    Аварийный шаттл покинул станцию. Через приблизительно { $eta ->
```
```suggestion
    Экстренный шаттл покинул станцию. Через приблизительно { $eta ->
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        [one] { $eta } минуту
+        [few] { $eta } минуты
+        *[many] { $eta } минут
+    } шаттл приоетит в { $dock_name }.
```
```suggestion
    } шаттл прилетит в { $dock_name }.
```
ъуъ

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    } шаттл приоетит в { $dock_name }.
+
+announce-emergency-shuttle-leaving-dock-example =
+    Аварийный спасательный шаттл вылетел из { $dock_name }.
```
```suggestion
    Экстренный спасательный шаттл вылетел из { $dock_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Аварийный спасательный шаттл вылетел из { $dock_name }.
+
+announce-emergency-shuttle-leaving-dock-demeter =
+    Аварийные шаттлы вылетают из RnCC Demeter и прибудут в течении { $eta ->
```
```suggestion
    Экстренные шаттлы вылетают из RnCC Demeter и прибудут в течении { $eta ->
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Evacuation Canceled
+
+announce-bluespace-distortion-cancels-evacuation =
+    Эвакуация была прервана из-за блюспейс искажения.
```
```suggestion
    Эвакуация была прервана из-за блюспейс-искажения.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Arrival Message
+
+announce-arrival-message-syndicate =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования.  
```
```suggestion
    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения.  
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
+
+announce-arrival-message-skipjack =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - выглядит невооруженной для приблизительного сканирования. 
```
```suggestion
    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения. 
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+
+announce-arrival-message-merchant =
+    Внимание, к станции приближается невооруженное грузовое судно, которое, по-видимому, является торговым судном.
```
```suggestion
    Внимание, к станции приближается невооруженное грузовое судно, по-видимому, являющееся торговым.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Departure Message
+
+announce-departure-message-syndicate =
+    Ваши посетители на пути к выходу из системы, удирая так быстро, как только можно. Скатертью дорожка.
```
```suggestion
    Ваши посетители на второй космической скорости покидают границы системы. Скатертью дорожка.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    Ваши посетители на пути к выходу из системы, удирая так быстро, как только можно. Скатертью дорожка.
+
+announce-departure-message-skipjack =
+    Ваши посетители на пути к выходу из системы, удирая так быстро, как только можно. Скатертью дорожка.
```
```suggestion
    Ваши посетители на второй космической скорости покидают границы системы. Скатертью дорожка.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Borer Spawn
+
+announce-borer-spawn =
+    Обнаружены неопознанные признаки жизни, поступающие с борта { $station_name }.
```
```suggestion
    Обнаружено присутствие неопознанных организмов на борту { $station_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Alien Spawn
+
+announce-alien-spawn =
+    Обнаружены неопознанные признаки жизни, поступающие с борта { $station_name }.
```
```suggestion
    Обнаружено присутствие неопознанных организмов на борту { $station_name }.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+    { $boss_name }
+
+announce-response-team-called =
+    Похоже, что для { $station_name } была запрошена группа экстренного реагирования.
```
```suggestion
    Похоже, что для { $station_name } был запрошен Отряд Быстрого Реагирования.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+
+announce-response-team-called =
+    Похоже, что для { $station_name } была запрошена группа экстренного реагирования.
+    Мы подготовим и отправим её как можно скорее.
```
```suggestion
    Мы подготовим и отправим его как можно скорее.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+## Response Team
+
+announce-response-team-call-failed =
+    Похоже, что для { $station_name } была запрошена группа экстренного реагирования.
```
```suggestion
    Похоже, что для { $station_name } был запрошен Отряд Быстрого Реагирования.
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        "гравитационных аномалии",
+        "червоточин в другое измерение",
+        "вспышек радиации",
+        "сверхматериальной пыль",
```
```suggestion
        "сверхматериальной пыли",
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+announce-response-team-disabled =
+    Из-за присутствия { PICK(
+        "политической нестабильности",
+        "квантовыъ флуктуации",
```
```suggestion
        "квантовых флуктуации",
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        "радикальных скреллианских трансэволюционеров",
+        "секретных операции по обеспечению безопасности"
+    ) } в регионе сковывает все доступные местные чрезвычайные ресурсы;
+    группы экстренного реагирования в настоящее время не могут быть вызваны, и
```
```suggestion
    Отряды Быстрого Реагирования в настоящее время не могут быть вызваны, и
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        "жуков-убийц, откладывающих яйца в шелуху живых",
+        "заброшенного транспорта, перевозящего образцы ксеноморфов",
+        "эмиссара гештальта, запрашивающего охрану",
+        "восстания таджарских рабов",
```
```suggestion
        "восстания рабов-таяран",
```

TobyThorne commented on localization/ru.ftl:
```diff
@@ -0,0 +1,738 @@
+        "эмиссара гештальта, запрашивающего охрану",
+        "восстания таджарских рабов",
+        "радикальных скреллианских трансэволюционеров",
+        "секретных операции по обеспечению безопасности"
```
```suggestion
        "секретных операции по обеспечению безопасности",
        "спецоперации по десиндикации Марса"
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    многие сотрудники либо уже мертвы, либо серьёзно пострадали. В настоящее время мы обнаруживаем все больше признаков того, что сама Вселенная начинает распадаться.
+
+    { $station_name }, самый большой источник помех точно указывает непосредственно на вас.
+    По нашим оценкам, у вас есть пять минут до того, как в ваших помещениях откроется разлом блюспейса.
```
```suggestion
    По нашим оценкам, у вас есть пять минут до того, как в ваших помещениях откроется пространственно-временной разлом.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    { $station_name }, самый большой источник помех точно указывает непосредственно на вас.
+    По нашим оценкам, у вас есть пять минут до того, как в ваших помещениях откроется разлом блюспейса.
+
+    Не существует известного способа остановить образование разлома, равно как и какого-либо способа избежать его.
```
```suggestion
    Не существует известного способа остановить его образование, равно как и какого-либо способа избежать.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-gateway-distress =
+    Обнаружена проблема в безопасности { $gateway_name }{ $location }. Попытка изолир... ОШИБКА.
+    IDS сообщает, что IPS был отключен. Уровень тревоги о вторжении: КРАСНЫЙ.
```
```suggestion
    система обнаружения сообщает, что система предотвращения вторжений была отключена. Уровень тревоги о вторжении: КРАСНЫЙ.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    Обнаружена проблема в безопасности { $gateway_name }{ $location }. Попытка изолир... ОШИБКА.
+    IDS сообщает, что IPS был отключен. Уровень тревоги о вторжении: КРАСНЫЙ.
+
+    Идентификатор сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
```
```suggestion
    Система обнаружения сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Gravity
+
+announce-gravity =
+    Всплеск обратной гравитационной связи обнаружен в системах массового распределения.
```
```suggestion
    Обратный гравитационный всплеск был замечен в системах распределения массы.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Power Restore
+
+announce-power-restore =
+    Все СМЕСы на { $station_name } были пополнены. Приносим извинения за причиненные неудобства.
```
```suggestion
    Все СМЕСы на { $station_name } были перезаряжены. Приносим извинения за причиненные неудобства.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    Экстренный шаттл был отозван.
+
+announce-emergency-shuttle-recall-demeter =
+    Наблюдающий отклонил экстренный запрос. Возвращайтесь на свои рабочие места.
```
```suggestion
    Экстренный запрос был отклонен наблюдающим. Возвращайтесь на свои рабочие места.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Arrival Message
+
+announce-arrival-message-syndicate =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения.  
```
```suggestion
    Внимание, мы зафиксировали приближение к вашей станции большого объекта, наши сканеры не зафиксировали вооружения.
```

TobyThorne replied:
Каво. В оригинале говорится как раз про поверхностное сканирование. Как в Космических Рейнджерах, когда у тебя сканнер пососный.

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-arrival-message-syndicate =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения.  
+    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
```
```suggestion
    Мы слишком далеко для перехвата - приготовьтесь к гостям.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
+
+announce-arrival-message-skipjack =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения. 
```
```suggestion
    Внимание, мы зафиксировали приближение к вашей станции большого объекта, наши сканеры не зафиксировали вооружения.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-arrival-message-skipjack =
+    Внимание, рядом с вами находится большая сигнатура, приближающаяся к станции - поверхностное сканирование не выявило наличие вооружения. 
+    Мы слишком далеко, чтобы её перехватить - приготовьтесь к посетителям.
```
```suggestion
    Мы слишком далеко для перехвата - приготовьтесь к гостям.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Departure Message
+
+announce-departure-message-syndicate =
+    Ваши посетители на второй космической скорости покидают границы системы. Скатертью дорожка.
```
```suggestion
    Ваши гости на второй космической скорости покидают границы системы. Скатертью дорожка.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    Ваши посетители на второй космической скорости покидают границы системы. Скатертью дорожка.
+
+announce-departure-message-skipjack =
+    Ваши посетители на второй космической скорости покидают границы системы. Скатертью дорожка.
```
```suggestion
    Ваши гости на второй космической скорости покидают границы системы. Скатертью дорожка.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Code Green
+
+announce-security-code-green-down =
+    Все угрозы станции миновали. Службы безопасности не может носить оружие на виду,
```
```suggestion
    Все угрозы станции устранены. Служба безопасности не может носить оружие на виду,
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-security-code-blue-up =
+    Станция получила достоверную информацию о возможной враждебной деятельности на станции.
+    У сотрудников службы безопасности может носить оружие на виду, разрешены "случайные обыски".
```
```suggestion
    Сотрудники службы безопасности могут носить оружие на виду, разрешены "случайные обыски".
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    У сотрудников службы безопасности может носить оружие на виду, разрешены "случайные обыски".
+
+announce-security-code-blue-down =
+    Непосредственная угроза миновала. Служба безопасности больше не может постоянно держать оружие наготове,
```
```suggestion
    Непосредственная угроза устранена.  Сотрудники службы безопасности больше не могут постоянно держать оружие в боевой готовности,
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-security-code-blue-down =
+    Непосредственная угроза миновала. Служба безопасности больше не может постоянно держать оружие наготове,
+    но может продолжать носить его на виду. Случайный поиск по-прежнему разрешен.
```
```suggestion
    но может продолжать носить его на виду. Случайные обыски по-прежнему разрешены.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-security-code-red-up =
+    Существует непосредственная серьезная угроза для станции.
+    Служба безопасности может постоянно носить оружие без кобуры.
```
```suggestion
    Сотрудники службы безопасности могут постоянно держать оружие в боевой готовности.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+announce-security-code-red-up =
+    Существует непосредственная серьезная угроза для станции.
+    Служба безопасности может постоянно носить оружие без кобуры.
+    Случайный поиск разрешен и рекомендуется.
```
```suggestion
    Случайные обыски по-прежнему разрешены и рекомендуются.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-security-code-red-down =
+    Механизм самоуничтожения был деактивирован, однако непосредственная серьезная угроза станции по-прежнему существует.
+    Служба безопасности может постоянно носить оружие без кобуры, разрешены и рекомендованы случайные обыски.
```
```suggestion
    Сотрудники службы безопасности могут постоянно держать оружие в боевой готовности, случайные обыски разрешены и рекомендуются.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Code Delta
+
+announce-security-code-delta-on =
+    Механизм самоуничтожения был включен.
```
```suggestion
    Механизм самоуничтожения был активирован.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-security-code-delta-on =
+    Механизм самоуничтожения был включен.
+    Весь экипаж проинструктирован выполнять все инструкции, данные руководителями персонала.
```
```suggestion
    Все члены экипажа должны подчиняться приказам руководителей персонала.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-response-team-call-failed =
+    Похоже, что для { $station_name } был запрошен Отряд Быстрого Реагирования.
+    К сожалению, в настоящее время мы не смогли отправить никого из её состава.
```
```suggestion
    К сожалению, в настоящее время мы не можем его отправить.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+## Self-destruct
+
+announce-self-destruct-terminal-stage =
+    Последовательность самоуничтожения достигла конечного обратного отсчета, системы прерывания были отключены.
```
```suggestion
    Механизм самоуничтожения достиг достиг точки невозврата, система аварийной отмены отключена.
```

NefrNerf commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-supply-beacon-deactivated =
+    Получен приоритетный запрос на поставку Nyx Rapid Fabrication #{ $x }-{ $y }.
+    Груз отправлен через баллистическую капсулу снабжения для немедленной доставки. Хорошего дня.
```
```suggestion
    Груз отправлен баллистической капсулой снабжения немедленной доставки. Хорошего дня.
```

Doster-d commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+
+announce-gateway-distress =
+    Обнаружена проблема в безопасности { $gateway_name }{ $location }. Попытка изолир... ОШИБКА.
+    система обнаружения сообщает, что система предотвращения вторжений была отключена. Уровень тревоги о вторжении: КРАСНЫЙ.
```
```suggestion
    Система обнаружения вторжения сообщает, что система предотвращения вторжений была отключена. Статус сигнала вторжении: КРАСНЫЙ.
```

Doster-d commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    Обнаружена проблема в безопасности { $gateway_name }{ $location }. Попытка изолир... ОШИБКА.
+    система обнаружения сообщает, что система предотвращения вторжений была отключена. Уровень тревоги о вторжении: КРАСНЫЙ.
+
+    Система обнаружения сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
```
```suggestion
    Система обнаружения вторжения сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
```

Doster-d commented on localization/ru.ftl:
```diff
@@ -0,0 +1,739 @@
+    система обнаружения сообщает, что система предотвращения вторжений была отключена. Уровень тревоги о вторжении: КРАСНЫЙ.
+
+    Система обнаружения сообщает, что передача была принята устройством. Источник передачи: "Неизвестен".
+    Подпись: "Сигнал бедствия NT высшего уровня". Обнаружен процесс триангуляции.
```
```suggestion
    Сигнатура: "Сигнал бедствия NT высшего уровня". Обнаружен процесс триангуляции.
```
https://ru.wikipedia.org/wiki/%D0%A1%D0%B8%D0%B3%D0%BD%D0%B0%D1%82%D1%83%D1%80%D0%B0_%D0%B0%D1%82%D0%B0%D0%BA%D0%B8

Commit:
add(translation): add basic translation


tweaks


Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/en.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
update


Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
Update localization/ru.ftl

Co-authored-by: goliath <kreeper3211225@gmail.com>
add gen defines tool and fix naming


Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Add script for checking vars between ftl files


Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
UP DLL


Apply suggestions from code review

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: TobyThorne <45202681+TobyThorne@users.noreply.github.com>
Apply suggestions from code review

Co-authored-by: NefrNerf <85663436+NefrNerf@users.noreply.github.com>
update dll


Update localization/ru.ftl

Co-authored-by: Doster <63081538+Doster-d@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: Doster <63081538+Doster-d@users.noreply.github.com>
Update localization/ru.ftl

Co-authored-by: Doster <63081538+Doster-d@users.noreply.github.com>

igorsaux commented:
☠️
